test-d/includes.ts

```diff
@@ -0,0 +1,35 @@
+import {Includes} from '..';
+import {expectType} from 'tsd';
```

Comment:
This should be first.

Reply:
Could you elaborate a bit? Do you mean:

```ts
import {expectType} from 'tsd';
import {Includes} from '..';
```

or

```ts
import {Includes} from '../ts41/includes';
import {expectType} from 'tsd';
```

Reply:
The former